src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/5.diff.sql

```diff
@@ -0,0 +1,668 @@
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
+
+IF NOT EXISTS 
+(
+    SELECT *
+    FROM    sys.columns
+    WHERE   NAME = 'TagValueUTC'
+            AND object_id = OBJECT_ID('dbo.ExtendedQueryTagDateTime')
+)
+BEGIN
+    ALTER TABLE dbo.ExtendedQueryTagDateTime
+    ADD TagValueUTC DATETIME2(7) NULL
+END
+
+CREATE UNIQUE CLUSTERED INDEX IXC_ExtendedQueryTagDateTime ON dbo.ExtendedQueryTagDateTime
+(
+    TagKey,
+    TagValue,
+    TagValueUTC,
+    StudyKey,
+    SeriesKey,
+    InstanceKey
+) WITH (DROP_EXISTING = ON)
+
+/*************************************************************
+    Table valued parameter to insert into Extended Query Tag table for data type Date Time.
+    V2 contains the TagValueUTC which separates it from V1.
+*************************************************************/
+IF TYPE_ID(N'InsertDateTimeExtendedQueryTagTableType_2') IS NULL
```

Comment:
Since all of this is running in a trasaction, do you need this IF?

Reply:
Theoretically this if check can be removed. However, there is a test that tries to apply the same diff twice to check the idempotency. This would fail claiming that the type "InsertDateTimeExtendedQueryTagTableType_2" already exists.